common/reviews/api/api-extractor.api.md

```diff
@@ -163,6 +168,12 @@ export class ExtractorResult {
     readonly warningCount: number;
 }
 
+// @public
+export interface IApiReportConfig {
+    fileName: string;
```

Comment:
It is slightly confusing that `fileName` includes the file extension but `IConfigApiReport.fileName` now excludes it.


Following [path.parse](https://nodejs.org/api/path.html#pathparsepath), maybe we can call it `baseFileName` or `baseName`:
> ![image](https://github.com/microsoft/rushstack/assets/4673363/23413916-1b28-4e66-9040-e8ff99c899c9)

Reply:
I might be overthinking this however. I was surprised to find that Heft's [ApiExtractorRunner.ts](https://github.com/microsoft/rushstack/blob/main/heft-plugins/heft-api-extractor-plugin/src/ApiExtractorRunner.ts) doesn't use these API's at all. I guess in most use cases API Extractor handles copying the files and printing warnings about report files. (?)

Reply:
Yeah, as far as I can tell that's strictly handled in `Extractor.ts`